applications/ecallmgr/src/ecallmgr_fs_authz.erl

```diff
@@ -338,5 +348,6 @@ rating_req(CallId, Props) ->
      ,{<<"Call-ID">>, CallId}
      ,{<<"Account-ID">>, AccountId}
      ,{<<"Direction">>, props:get_value(<<"Call-Direction">>, Props)}
+     ,{<<"Send-Empty">>, 'true'}
```

Comment:
@jamesaimonetti thanks, i misinterpreted because of the way github presented the changes. it seemed it was part of set_rating_ccvs.